search.ipynb

```diff
@@ -1754,74 +1002,44 @@
     "    You need to specify the h function when you call astar_search, or\n",
     "    else in your Problem subclass.\"\"\"\n",
     "    h = memoize(h or problem.h, 'h')\n",
-    "    iterations, all_node_colors, node = best_first_graph_search(problem, lambda n: n.path_cost + h(n))\n",
+    "    iterations, all_node_colors, node = best_first_graph_search(problem, \n",
+    "                                                                lambda n: n.path_cost + h(n))\n",
     "    return(iterations, all_node_colors, node)\n"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 35,
-   "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "application/vnd.jupyter.widget-view+json": {
-       "model_id": "c53e9003385741398e11639b52bdf53a",
-       "version_major": 2,
-       "version_minor": 0
-      },
-      "text/plain": [
-       "A Jupyter Widget"
-      ]
-     },
-     "metadata": {},
-     "output_type": "display_data"
-    },
-    {
-     "data": {
-      "application/vnd.jupyter.widget-view+json": {
-       "model_id": "f8a00ed468174908b387f6f2b74e46a5",
-       "version_major": 2,
-       "version_minor": 0
-      },
-      "text/plain": [
-       "A Jupyter Widget"
-      ]
-     },
-     "metadata": {},
-     "output_type": "display_data"
-    }
-   ],
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
    "source": [
     "all_node_colors = []\n",
     "romania_problem = GraphProblem('Arad', 'Bucharest', romania_map)\n",
-    "display_visual(romania_graph_data, user_input = False, algorithm = astar_search, problem = romania_problem)"
+    "display_visual(romania_graph_data, user_input = False, \n",
```

Comment:
Remove the spaces around the equal signs ( '=' ).

This issue is present in following cells too.